test/integration/api/workflow.spec.js

```diff
@@ -24,6 +24,125 @@ let userWebhookUrl
 
 const defaultTestDelay = 4000
 
+let createdWorkflows
+
+async function createWorkflowLogs (t) {
+  if (createdWorkflows) return createdWorkflows
+
+  const authorizationHeaders = await getAccessTokenHeaders({
+    t,
+    permissions: [
+      'workflowLog:list:all',
+      'workflow:create:all',
+      'category:create:all',
+      'entry:create:all',
+      'message:create:all',
+    ]
+  })
+
+  // should create workflows that listen to events that are not triggered by any tests below
```

Comment:
Why not integrate assertions related logs to existing workflow tests?

If you can’t and you really need such a full scenario and isolation from other workflow tests you should maybe create another  `workflowLog.spec.js` test file.

Reply:
I integrated the logs assertion into existing workflows.
I also put workflow/webhook logs into fixtures to be able to test filters in dedicated test.

Reply:
Thanks but you are not answering my question, and it’s better to discuss costly changes before making them.

Reply:
Indeed, that's not a bad idea to separate workflow/webhook logs into their own test file.
Fixtures are reverted as they were and logs are generated at the start of test suite.